3rdparty/zlib-ng/CMakeLists.txt

```diff
@@ -1403,8 +1403,4 @@ if(ENABLE_SOLUTION_FOLDERS)
   set_target_properties(${ZLIB_INSTALL_LIBRARIES} PROPERTIES FOLDER "3rdparty")
 endif()
 
-if(NOT BUILD_SHARED_LIBS)
-  ocv_install_target(${ZLIB_INSTALL_LIBRARIES} EXPORT OpenCVModules ARCHIVE DESTINATION ${OPENCV_3P_LIB_INSTALL_PATH} COMPONENT dev)
-endif()
```

Comment:
It breaks distribution with self-built zlib-ng.

Reply:
This is moved to cmake/OpenCVFindLibsGrfmt.cmake which means to install target when building or finding zlib. At first, I didn't touch these lines. However, there is a CI check fails without these lines. I don't remember which check now, sorry.